a4-speakup/assets/scss/components/_blocks.scss

```diff
@@ -28,3 +28,12 @@
 .block__header_content {
     padding: 2*$spacer 0;
 }
+
+.block__background {
+    .rich-text {
+        a {
+            color: $white;
+            text-decoration: underline;
+        }
```

Comment:
I know this is nested but i'm not sure how else to style wagtail stuff?
rich-text nest is needed so cta button does not get underlined

Reply:
The nesting is fine, when we have to be that specific to style other libraries classes! Just when we write the css and the html, we try not to do that because components and reusability!